cmd/ttn-lw-cli/commands/end_device_templates.go

```diff
@@ -37,6 +37,13 @@ func templateFormatIDFlags() *pflag.FlagSet {
 	return flagSet
 }
 
+func endDeviceProfileIDFlags() *pflag.FlagSet {
```

Comment:
This is inconsistent with the field naming in the request message. The field is called `end-device-profile-ids`, not `vendor-ids`.

You don't need to manually write this flag set creator. If you add
```
  option (thethings.flags.message) = { set: true };
```
to the `EndDeviceProfileIdentifiers` and `ConvertEndDeviceTemplateRequest` messages, both this and the `templateFormatIDFlags` will be generated for you. See the resulting `AddSetFlagsForConvertEndDeviceTemplateRequest` function `pkg/ttnpb`.

Also consider using `ConvertEndDeviceTemplateRequest.SetFromFlags` instead of `getEndDeviceProfileID`.

Reply:
I ended only applying this to the `EndDeviceProfileIdentifiers`, the reason for this is that when loading the values of `ConvertEndDeviceTemplateRequest` the slice of bytes assigned to the `data` field is expected but that is fetched through a `--local-file` flag which read the local file. [code here](https://github.com/TheThingsNetwork/lorawan-stack/blob/5b37ddd1e6bc9a86f07bf8a43748ad96500968e2/cmd/ttn-lw-cli/commands/flags.go#L237-L250)